test/units/testsuite-73.sh

```diff
@@ -648,6 +648,40 @@ EOF
     assert_in "XKBLAYOUT=us" "$output"
 }
 
+locale_gen_cleanup() {
+    if mountpoint -q /usr/lib/locale; then
+        rm -fr /usr/lib/locale/*
```

Comment:
Hm? Why necessary?

Reply:
Various running apps keep the mountpoint busy:

```
++ locale_gen_cleanup
++ mountpoint -q /usr/lib/locale
++ umount /usr/lib/locale
umount: /usr/lib/locale: target is busy.
```

```
# lsof | grep /usr/lib/locale
agetty    288                          root  mem       REG              254,2              574367 /usr/lib/locale/locale-archive (path dev=0,39, inode=3)
agetty    288                          root  mem       REG              254,2              527694 /usr/lib/locale/C.UTF-8/LC_CTYPE (stat: No such file or directory)
bash      544                       vagrant  mem       REG              254,2              574367 /usr/lib/locale/locale-archive (path dev=0,39, inode=3)
sudo      549                          root  mem       REG              254,2              574367 /usr/lib/locale/locale-archive (path dev=0,39, inode=3)
su        550                          root  mem       REG              254,2              574367 /usr/lib/locale/locale-archive (path dev=0,39, inode=3)
bash      551                          root  mem       REG              254,2              574367 /usr/lib/locale/locale-archive (path dev=0,39, inode=3)
lsof      748                          root  mem       REG               0,39  3415040          3 /usr/lib/locale/locale-archive
grep      749                          root  mem       REG               0,39  3415040          3 /usr/lib/locale/locale-archive
lsof      750                          root  mem       REG               0,39  3415040          3 /usr/lib/locale/locale-archive
```

Removing the stuff before the umount call seems to help (albeit it's not very pretty).

Reply:
Also, lazy unmount seems to work as well, which might be a bit prettier.